data/assets/scene/digitaluniverse/allsky_hydrogenalpha.asset

```diff
@@ -0,0 +1,68 @@
+local textures = asset.resource({
+  Name = "Multiwavelength Milky Way Textures",
+  Type = "HttpSynchronization",
+  Identifier = "digitaluniverse_multiwavelength_milkyway_textures",
+  Version = 1
+})
+
+
+local Object = {
+  Identifier = "AllSky_HAlpha",
+  Transform = {
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = { 0, 0, math.pi }
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphereImageLocal",
+    Enabled = false,
+    Size = 9.2E21,
+    Segments = 40,
+    Opacity = 0.4,
+    Texture = textures .. "mwHalpha-f.png",
+    Orientation = "Inside",
+    MirrorTexture = true,
+    FadeOutThreshold = 0.025,
+    Background = true
```

Comment:
```suggestion
    FadeOutThreshold = 0.025
```

Or is that supposed to be: `RenderBinMode = "Background"`?

Reply:
Good catch.  This can be removed. It was simply copied over from the old file and I've traced this line back to at least 5 years in the code base, and even longer for the milky way code. 

Not sure how this property worked at that point, or if it ever worked. But by now I think we can remove it :)